### PR TITLE
PIL-1076 App appearance selector

### PR DESCRIPTION
**Pull request by divyaBacancy**
https://user-images.githubusercontent.com/66996635/136950653-e7700565-b4c6-4907-857f-876ecee26d4f.mov

### CODE REVIEW

**Comment by IAmKio** [comment]
Hey @divyaBacancy - the design has been slightly updated here to reflect the latest design:
https://www.sketch.com/s/55f2c851-6474-46d7-b811-c9c4e7a1d44e/a/rbeGyD7

Could you update the design please?

**Comment by divyaBacancy** [comment]
AppAppearence Dark Theme

File: src/screens/AppAppearence/AppAppearence.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    dispatch(setAppThemeAction(LIGHT_THEME, true));
+  };
+
+  const onPressDarkTheme = () => {
+    setDarkThemePressed(true);
+    setLightThemePressed(false);
+    dispatch(setAppThemeAction(DARK_THEME, true));
+  };
+
+  const style = {
```
Comment (IAmKio) [inline src/screens/AppAppearence/AppAppearence.js:68]: `style` is too generic here, it took me a while to see where it was being used. Could you change this to something more specific please?

File: src/screens/AppAppearence/AppAppearence.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            </Text>
+          </Themes>
+        </ThemeView>
+        <Button title={t('auth:button.confirm')} size="large" style={styles.confirmButton} />
+      </Center>
+    </Container>
+  );
+};
+
+const styles = {
```
Comment (IAmKio) [inline src/screens/AppAppearence/AppAppearence.js:102]: `styles` - the same as above, could you make this a bit more specific please?

File: src/screens/AppAppearence/AppAppearence.js (hunk 1 above)
Comment (IAmKio) [reply src/screens/AppAppearence/AppAppearence.js:68]: Also if you could move this to where the other style definitions are please.

**Review by IAmKio — CHANGES_REQUESTED** [review]
Some minor changes but overall, good work!

File: src/screens/AppAppearence/AppAppearence.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+// Utils
+import { fontStyles, spacing, appFont } from 'utils/variables';
+import { getThemeColors } from 'utils/themes';
+
+// Actions
+import { setAppThemeAction } from 'actions/appSettingsActions';
+
+const lightTheme = require('assets/images/appAppearence/lightTheme.png');
+const darkTheme = require('assets/images/appAppearence/darkTheme.png');
```
[the remark this answers is not in the document]
Comment (divyaBacancy) [reply src/screens/AppAppearence/AppAppearence.js:46]: Done

File: src/screens/AppAppearence/AppAppearence.js (hunk 2 above)
Comment (divyaBacancy) [reply src/screens/AppAppearence/AppAppearence.js]: Done

File: src/screens/AppAppearence/AppAppearence.js (hunk 1 above)
Comment (divyaBacancy) [reply src/screens/AppAppearence/AppAppearence.js]: Done

File: src/screens/AppAppearence/AppAppearence.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    setDarkThemePressed(true);
+    setLightThemePressed(false);
+    dispatch(setAppThemeAction(DARK_THEME, true));
+  };
+
+  const selectedThemeStyle = {
+    borderColor: colors.buttonPrimaryBackground,
+    borderWidth: 2,
+    borderRadius: 10,
+  };
```
Comment (IAmKio) [inline src/screens/AppAppearence/AppAppearence.js:73]: Please remember to move this down to the other style definitions (around line 106)

**Review by IAmKio — APPROVED** [review]
LGTM 👍

**Pull request merged**